src/Backend/Command.ts

```diff
@@ -37,7 +37,7 @@ class Command extends Array<string> {
   }
 
   str(): string {
-    return this.strs().join(' ');
+    return (this.root ? 'sudo ' : '') + this.strs().join(' ');
```

Comment:
(optional) Just another thought. It's not a normal case but what if the user is `root`...
maybe add a `TODO: support or deny root account`

Reply:
At this point, it is impossible to handle it because it is not known which environment to call. However, when we actually run this command, we can consider how to ignore or deny root command.